lib/knackhq/client.rb

```diff
@@ -48,10 +48,9 @@ def records_by_page(object, page_number)
                      .records
                      .get(:params => { :page => page_number,
                                        :rows_per_page => 500 })
-                     .to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:records]
-      symbolize_hash_keys!(transform_hash_keys)
+      payload = payload_hash(hash_request)
+      return [] if payload.empty?
+      payload[:records]
     end
 
     def records_info(object)
```

Comment:
But the Knach API is still performing the same query? And the non info version still returns the same info as well as the objects?